internal/common/metrics/utils.go

```diff
@@ -79,9 +79,9 @@ func GetMetricsScopeForWorkflow(ts tally.Scope, workflowType string) tally.Scope
 }
 
 // GetMetricsScopeForRPC return properly tagged tally scope for workflow execution
-func GetMetricsScopeForRPC(ts tally.Scope, workflowType, activityType, taskqueueName string) tally.Scope {
+func GetMetricsScopeForRPC(ts tally.Scope, workflowType, activityType, taskQueueName string) tally.Scope {
```

Comment:
Not directly related to this change, but I noticed the casing on this method was inconsistent with other places in the code so I made that minor tweak while I was in there.

Reply:
Great.